templates/services/parallel_service_orchestration/src/parallel_service_orchestration/tests/main_test.bal

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Let's delete the `main_test.bal` since no usage.

Reply:
Done